src/dotnet/APIView/APIViewWeb/Exceptions/DuplicateLineIdException.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+
+namespace APIViewWeb.Exceptions
+{
+    public class DuplicateLineIdException : Exception
+    {
+        public string Language { get; }
+        public string DuplicateLineId { get; }
+
+        public DuplicateLineIdException(string language, string duplicateLineId) 
+            : base($"API review generation failed due to a language parser error. " +
+                   $"The parser generated duplicate line identifiers (IDs: '{duplicateLineId}'), which indicates " +
+                   $"an issue in the language-specific parser for {language}.")
```

Comment:
I'd recommend a message like "APIView unexpectedly received duplicate line identifiers (IDs: ...). Please contact the developer of the {langauge} APIView parser."